src/Sylius/Bundle/LocaleBundle/Twig/LocaleExtension.php

```diff
@@ -50,6 +91,52 @@ public function getCurrentLocale()
     }
 
     /**
+     * Returns current language direction.
+     *
+     * @return string Possible values are `rtl` and `ltr`
+     */
+    public function getCurrentDirection()
+    {
+        return $this->helper->getCurrentDirection();
+    }
+
+    /**
+     * Checks whether if current language direction is RTL or not.
+     *
+     * @return bool True if language direction is RTL
+     */
+    public function isRtl()
+    {
+        return $this->helper->getCurrentDirection() === 'rtl';
```

Comment:
I would recommend an *yoda style* check. `'rtl' ===  $this->helper->getCurrentDirection() `